hs_core/management/commands/check_irods_files.py

```diff
@@ -61,17 +61,29 @@ def add_arguments(self, parser):
 
         # Named (optional) arguments
         parser.add_argument(
-            '--repair',
+            '--sync_ispublic',
             action='store_true',  # True for presence, False for absence
-            dest='repair',
-            help='check for local unreferenced iRODS files',
+            dest='sync_ispublic',
+            help='delete unreferenced iRODS files',
```

Comment:
@alvacouch The help text above does not seem right for the argument, a copy and paste issue?